pulp/tests/test_pulp.py

```diff
@@ -149,7 +149,7 @@ def test_pulp_012(self):
             # CPLEX_DLL Does not report unbounded problems, correctly
             # GUROBI_CMD has a very simple interface
             pulpTestCheck(prob, self.solver, [const.LpStatusNotSolved])
-        elif self.solver.__class__ in [PULP_CHOCO_CMD, CHOCO_CMD]:
+        elif self.solver.__class__ in [PULP_CHOCO_CMD, CHOCO_CMD, SCIP_CMD]:
```

Comment:
wouldn't SCIP_CMD return `const.LpStatusNotSolved` ? if this is the case, better to put it with `CPLEX_DLL` and `GUROBI_CMD` above. Or on some other if clause that matches SCIP's output status.

Reply:
Yes my mistake